python/lsst/ts/externalscripts/make_love_stress_tests.py

```diff
@@ -0,0 +1,341 @@
+        - LOVE host location
+        - Number of clients
+        - Number of messages to store
+        - CSCs
+
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Configuration with several attributes, defined in `get_schema`
+
+        Notes
+        -----
+        Saves the results on several attributes:
+
+        * location : a string, with the location of the LOVE instance to stress
+        * config : a dict, with the number_of_clients and number
+            of_messages configurations
+        * remotes : a dict, with each item as
+            CSC_name[:index]: `lsst.ts.salobj.Remote`
+
+        Constructing a `salobj.Remote` is slow (DM-17904), so configuration
+        may take a 10s or 100s of seconds per CSC.
+        """
+        self.log.info("Configure started")
+
+        # set configurations
+        self.location = config.location
+        self.config = {
+            "number_of_clients": config.number_of_clients,
+            "number_of_messages": config.number_of_messages,
+        }
+
+        # get credentials
+        self.username = "user"
+        self.password = os.environ.get("USER_USER_PASS")
+        if self.password is None:
+            raise RuntimeError(
+                "Configuration failed: environment variable USER_USER_PASS not defined"
+            )
+
+        # construct remotes
+        remotes = dict()
+        for name_index in config.data:
+            name, index = salobj.name_to_name_index(name_index)
+            self.log.debug(f"Create remote {name}:{index}")
+            if name_index not in remotes:
+                remote = salobj.Remote(domain=self.domain, name=name, index=index)
+                remotes[name_index] = remote
+        self.remotes = remotes
+
+    async def run(self):
+        """Run script."""
+        tasks = [
+            remote.start_task
+            for remote in self.remotes.values()
+            if not remote.start_task.done()
+        ]
+
+        if tasks:
+            self.log.info(
+                f"Waiting for {len(self.remotes.values())} remotes to be ready"
+            )
+            await asyncio.gather(*tasks)
+
+        # Enable all CSCs
+        for remote in self.remotes.values():
+            await salobj.set_summary_state(remote=remote, state=salobj.State.ENABLED)
+
+        event_streams = dict()
+        telemetry_streams = dict()
+        for name_index in self.remotes:
+            name, index = salobj.name_to_name_index(name_index)
+            salinfo = salobj.SalInfo(self.domain, name)
+            try:
+                event_streams[name_index] = salinfo.event_names
+                telemetry_streams[name_index] = salinfo.telemetry_names
+            finally:
+                await salinfo.close()
+
+        # Create clients and listen to ws messages
+        self.log.info(
+            f"Waiting for {self.config['number_of_clients']} Manager Clients to be ready"
```

Comment:
If you make self.config a copy of config (as suggested above) this will be:
```
f"Waiting for {self.config.number_of_clients} Manager Clients to be ready"
```